### PR TITLE
Number workflows

### PR DESCRIPTION
**Pull request by chantelleleveille**
# Main Goal: Update workflow names to indicate which figure in the paper they generate. 

### For review:
- please confirm if you like this numbering/naming convention
- @cfrick13  Please double check s2 and s3 and s18. I did not rename your base workflow files because I wasn't 100p sure what came from where!
- s17 I think is a code free figure? or can you confirm what workflow it is associated with. 

### Bonus one line bug fix: 
- Full tracks dataframe was not sorted by index sequence and it was causing plots to look crazy! 

-----

All workflows succeeded!

Well except these two but they fail for known reasons (new density metric needs to be updated) and will be addressed in a different PR! 

```
figure_s12_feeding_control (178.8s) (23.6GB). KeyError "['2d_area_nuc_cell_ratio'] not in index". See stack trace above.
figure_s6_s16_inhibitors (92.0s) (10.7GB). KeyError "['2d_area_nuc_cell_ratio'] not in index". See stack trace above.
```

### CODE REVIEW

File: nuc_morph_analysis/lib/preprocessing/filter_data.py [hunk 1]
```diff
@@ -939,6 +939,7 @@ def all_timepoints_full_tracks(df):
     """
     df = all_timepoints_minimal_filtering(df)
     df = df.loc[df["is_full_track"] == True]
+    df = df.sort_values('index_sequence')
```
Comment (jcass11) [inline nuc_morph_analysis/lib/preprocessing/filter_data.py:942]: Thank you!

**Comment by jcass11** [comment]
- Checked all the new referenced fig numbers and they all match the right figures ✅ 
- Naming convention and organization all looks good to me
- Yes I believe S17 is code-free
- Thanks for the bug fix!

**Review by jcass11 — APPROVED** [review]
Sounds like @chantelleleveille is going to run all workflows so I am approving without running. Thank you!

**Review by pgarrison — APPROVED** [review]
Nice!!

**Pull request merged**